api/tests/Api/SnapshotTests/__snapshots__/ResponseSnapshotTest__testRootEndpointMatchesSnapshot__1.json

```diff
@@ -21,7 +21,7 @@
             "templated": true
         },
         "categories": {
-            "href": "\/categories{\/id}{?camp,camp[]}",
+            "href": "\/categories{\/id}{\/action}{?camp,camp[]}",
```

Comment:
why do we have this change?
and why can't i see the /camps/{\/id}/categories/{\/id} route?

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull with commit https://github.com/ecamp/ecamp3/pull/4968/commits/ed8433db1f9471e85a0c455247980fc504648282

I will update the snapshots here, as soon as this commit is merged into this PR.